runtime/parser/declaration.go

```diff
@@ -1384,6 +1384,7 @@ func parseMemberOrNestedDeclaration(p *parser, docString string) (ast.Declaratio
 				staticPos,
 				nativePos,
 				identifier,
+				docString,
```

Comment:
This and the changes below are from another PR. To avoid a re-review, please update the target branch first to include these changes (i.e. merge `feature/stable-cadence` into `feature/capability-controllers`)